### DISCUSSION TITLE
Adding a ready-to-deploy Flask example

### DISCUSSION THREAD

**Discussion by natanweinberger** [vercel/vercel#4620] [Ideas] [answered]
I'd like to contribute a ready-to-deploy Flask example.

I was looking for one earlier and didn't find one in the /examples directory. I ended up doing some trial-and-error before I successfully got a Flask app working. I think a working, zero-configuration example would be really helpful!

This demo is roughly what I have in mind: [https://github.com/natanweinberger/vercel-flask](https://github.com/natanweinberger/vercel-flask).

Please let me know what you think! I would be glad to open a PR and add this example if the maintainers agree its useful.

**Comment by styfle** [comment, marked as answer]
Thanks for the suggestion!

We are not accepting examples for Flask at this time.

The official examples are reserved for [zero config](https://vercel.com/blog/zero-config) frontend frameworks such as Next.js, Gatsby, Hugo, etc that do not use a `vercel.json` config.
